ToDoList/Views/AddTaskSheetView.swift

```diff
@@ -8,12 +8,34 @@
 import SwiftUI
 
 struct AddTaskSheetView: View {
-    @Binding var taskDescription: String
-    @Binding var taskDate: Date
-    @Binding var taskPriority: String
-    @Binding var reminderEnabled: Bool
-    var onSave: () -> Void
+    @Environment(\.dismiss) var dismiss
+    @ObservedObject var viewModel: TaskViewModel
+    var taskToEdit: Task?
```

Comment:
```suggestion
   @Binding var taskToEdit: Task?
```

Then you don't have to fill in the values in the init, you can pass in the task where you're calling this view